spec/features/normalized_download_spec.rb

```diff
@@ -56,5 +56,25 @@
         expect(marc['900']['b']).to eq organization.code
       end
     end
+
+    it 'tracks the download' do
+      visit organization_url(organization)
+
+      expect do
+        # Note, this won't work in a driver other that Rack::Test w/o some other magic
+        click_link "#{organization.slug}-2020-01-01-marc21.mrc.gz"
+      end.to change(Ahoy::Event, :count).by(1)
+
+      expect(Ahoy::Event.last.properties.with_indifferent_access).to include(
+        attachment_name: 'full_dump_binary',
```

Comment:
Is there a way to query this somehow through the expected attachment/file ?

Or can we only do this from the visit itself?

Reply:
I don't think ahoy gives us anything for free, but I /think/ we can use the postgres json query stuff to dig it out easily (and, if not... just run a daily statistics report and call it good? 🤷‍♂️).

Reply:
WFM